server/plugin/utils.go

```diff
@@ -38,8 +44,17 @@ func (p *Plugin) validateTime(time string) string {
 	timeRegex := regexp.MustCompile(`^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$`)
 	timeMatched = timeRegex.FindAllStringSubmatch(time, -1)
 	if timeMatched == nil {
-		return TimeValidationError
+		return constants.TimeValidationError
 	}
 
 	return ""
 }
+
+func (p *Plugin) IsCharCountSafe(attachments []*model.SlackAttachment) bool {
+	bytes, err := json.Marshal(attachments)
+	if err != nil {
+		p.API.LogDebug("Error in marshaling the attachments", "Error", err.Error())
```

Comment:
should not we return here in case of error?

Reply:
I don't even think that there's a need to handle the error here, so reverting this.